charts/passbolt-operator/templates/deployment.yaml

```diff
@@ -74,10 +103,6 @@ spec:
                   name: {{ include "passbolt-operator.secret.name" . }}
                   key: gpg_key
           {{- end }}
-          ports:
-            - name: http
-              containerPort: 8081
-              protocol: TCP
```

Comment:
Just curious: where was this port used before? I didn't find anything 🤔

Reply:
It wasn't used at all.